packages/react-server/src/ReactFizzServer.js

```diff
@@ -2456,3 +2484,18 @@ export function abort(request: Request, reason: mixed): void {
     fatalError(request, error);
   }
 }
+
+// If new resources are identified out of band with a task we can still potentially
+// emit them early by pinging them.
+export function pingRequest(): void {
```

Comment:
Too generic. Unclear what this will mean when it changes.

There are optimizations we can do if we assume that this is resources.

`flushResources()` maybe? It's not really a sync flush but close enough.